src/Greetingandintro/Greeting.jsx

```
@@ -0,0 +1,749 @@
+        answer: 1,
+        audioIndex: 2
+      }
+    ]
+  }
+];
+
+export default function GreetingsCourse() {
+  const [currentLessonIdx, setCurrentLessonIdx] = useState(0);
+  const [currentExIdx, setCurrentExIdx] = useState(0);
+  const [feedback, setFeedback] = useState(null);
+  const [answers, setAnswers] = useState([]);
```

ESLint

[error] 'answers' is assigned a value but never used.

(no-unused-vars)